pdi-step-plugin-archetype/src/main/resources/archetype-resources/src/main/java/__plugin_class_name__Data.java

```diff
@@ -24,14 +24,17 @@
 import org.pentaho.di.trans.step.BaseStepData;
 import org.pentaho.di.trans.step.StepDataInterface;
 
+public class $ {
```

Comment:
this ${plugin_class_name} is a variable that gets substituted. it shouldn't be split across lines.